GettingStarted.md

```diff
@@ -22,10 +22,10 @@ For some of the supported package managers and SCMs additional tools need to be
 
 * CVS
 * Mercurial
-* NPM (required for this tutorial because we scan `mime-types` which is an NPM project)
+* Node JS version 8.x and NPM version 5.5.1 (required for this tutorial because we scan `mime-types` which is an NPM project)
```

Comment:
Nits:
- Looks like "Node.js" is the official spelling, so we should use that.
- I feel we could drop the "version" words here and just say "NPM 5.5.1 and Node.js 8.x". I'm also swapping the order of NPM and Node.js here, as what counts for us is the version of NPM; Node.js is more or less implied.